runtime/drivers/clickhouse/model_manager.go

```diff
@@ -43,8 +43,8 @@ type ModelOutputProperties struct {
 	SampleBy string `mapstructure:"sample_by"`
 	// TTL sets ttl for column and table.
 	TTL string `mapstructure:"ttl"`
-	// Settings set the table specific settings.
-	Settings string `mapstructure:"settings"`
+	// TableSettings set the table specific settings.
+	TableSettings string `mapstructure:"table_settings"`
```

Comment:
NOTE : This is a breaking change but should be fine. Don't think there are projects using `settings` property.